graphql2/schema.graphql

```diff
@@ -655,6 +657,13 @@ input RotationSearchOptions {
   favoritesFirst: Boolean = false
 }
 
+input NextRotationHandoffTimesInput {
+  start: ISOTimestamp!
+  timeZone: String!
```

Comment:
What do you think of making this field optional, defaulting to the timeZone of the rotation as output?

Reply:
We don't have a rotation (no ID) as this is a top-level query

Reply:
Ah I see, I think that's what has been confusing me. When I see a query of `NextRotationHandoffTimesInput`, I expect to give it (or have available) a rotation ID. `Next` also sounds to me like there is also a rotation that already exists, and we are getting the next vals. I think different verbiage like `Calculate` or `Anticipate` would be more appropriate

Reply:
`calcRotationHandoffTimes` and `CalcRotationHandoffTimesInput` maybe?